### PR TITLE
Add worker-cli documentation

### PR DESCRIPTION
**Pull request by ehl-jf**

### CODE REVIEW

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# CLI for JFrog Platform Services
+
+This page describes how to use JFrog CLI with JFrog Platform Services.
+
+Read more about JFrog CLI [here](https://jfrog-external.fluidtopics.net/r/help/JFrog-CLI/JFrog-CLI).
+
+**Managing platform Workers**
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/README.md:7]: remove "platform"

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-add-secret.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Initialize a Worker
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-add-secret.md:1]: title is not correct

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-deploy.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Initialize a Worker
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-deploy.md:1]: same title error

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-deploy.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## Overview
+
+This command is used to Deploy a worker.
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-deploy.md:5]: lower case, "deploy". Maybe say "This command is used to update the worker definition (code, description , filter, secret ...) on your Artifactory instance

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-execute.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# Execute a manual worker
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-execute.md:1]: Align, either "manual" or "generic"

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-dry-run.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# Test a worker
+
+## Overview
+
+Dry run a worker. The worker needs to be initialized before running this command.
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-dry-run.md:5]: add link to init. I think this statement is true for most of the commands, please add it to all command where it's needed

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-init.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
+# Initialize a Worker
+
+## Overview
+
+This command is used to initialize a new platform worker.
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-init.md:5]: be more descriptive like "This command will create all necessary file in the current directory in order to start writing your worker : sample code, manifest file and sample tests" something like that

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-list-event.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+# Initialize a Worker
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-list-event.md:1]: title

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-list-event.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+## Overview
+
+This command list all the available events for a given server.
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-list-event.md:5]: "on your Artifactory"

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-list.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# List Workers
+
+## Overview
+
+List workers. The default output is a CSV format with columns `name`,`action`,`description`,`enabled`.
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-list.md:5]: List workers saved on your Artifactory instance.

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-undeploy.md [hunk 11]
```diff
@@ -0,0 +1,23 @@
+# Initialize a Worker
+
+## Overview
+
+This command is used to initialize a new platform worker.
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-undeploy.md:5]: comment is bad

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-undeploy.md [hunk 12]
```diff
@@ -0,0 +1,23 @@
+# Initialize a Worker
```
Comment (RemiBou) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-undeploy.md:1]: title

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-add-secret.md [hunk 13]
```diff
@@ -0,0 +1,22 @@
...
+| `--edit`               | \[Default: false] Whether to update an existing secret. |
+| **Command arguments:** |                                                         |
+| secret-name            | The secret name                                         |
+
+## Example
+
+Add a secret name `my-secret` to a worker initialized in the current directory.
+
+```
+jfrog worker add-secret my-secret
```
Comment (eyalbe4) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-add-secret.md:21]: Non of the existing examples in the JFrog CLI documentation use the `jfrog` executable. To maintain consistency and avoid confusion, you should the `jf` executable all across.

File: jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/README.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
+# CLI for JFrog Platform Services
+
+This page describes how to use JFrog CLI with JFrog Platform Services.
```
Comment (eyalbe4) [inline jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/README.md:3]: I'm not sure that the term "Platform Services" would be clear for the user. You can consult with s technical writer about the best introductory phrasing. 
There's a chance some of the phrasing throughout this PR can be further improved.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

**Pull request merged**